### PR TITLE
Checkins

### PR DESCRIPTION
**Pull request by charlesx2013**
This is a basic checkin - no user type validations yet.

Pls @SamLau95

### CODE REVIEW

**Comment by SamLau95** [comment]
No.

**Comment by charlesx2013** [comment]
Also pls @chenhoward

File: app/controllers/api/v1/check_ins_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  def show
+    render json: @check_in, serializer: CheckInSerializer
+  end
+
+  def create
+    if @check_in.save
+      render json: @check_in, serializer: CheckInSerializer
+      error_response(@check_in)
```
Comment (chenhoward) [inline app/controllers/api/v1/check_ins_controller.rb]: I might be confused but I would imagine this would be in the else.

Comment (charlesx2013) [inline app/controllers/api/v1/check_ins_controller.rb]: whoops

**Comment by chenhoward** [comment]
Instead of doing render json: blah blah blah you can just set the default response type of the route:

http://guides.rubyonrails.org/routing.html#defining-defaults

**Comment by charlesx2013** [comment]
screen shot 2015-10-01 at 1 21 27 pm

"LGTM" - @chenhoward

**Pull request merged**